app/styles/app.scss

```diff
@@ -748,3 +748,18 @@ hr {
     background-size: cover;
     background-image: url("img/logo_ua.png");
 }
+
+.validated-input .input-error {
```

Comment:
If possible, don't add component styles to app.scss.

We prefer "pod style components": they should live in a folder `componentName`, with two files:
`component.js`
`template.hbs`

See example: https://github.com/abought/ember-osf/tree/4f2947e7a0c3622b8f1c21d89e321caadf919b4e/addon/components/file-browser

If you have custom styles, then to keep the component together in a single concise package, we prefer those go in the same folder, in a file called `style.scss`. You can generate a new pod component at any time using a command line flag like so:
`ember generate component myComponentIsNamedBob --pod`

This makes it easier to move and remove components, so that we minimize leftover cruft lying around in other folders. For now, just renaming the folder and files would be sufficient to make it magically work.